README.md

```diff
@@ -22,31 +22,39 @@ We have a fantastic community growing on Discord. Click [here](https://discord.g
 2. Clone with `git clone https://github.com/YOUR_USERNAME/codu.git`.
 3. Navigate to the project directory `cd codu`.
 4. Install dependencies with:
+
 ```bash
 npm install
 ```
-  - If you have any issues installing dependencies, check your node version against that defined in the `.nvmrc`. If you're using `nvm`, you can run `nvm use` before installing dependencies.
+
+- If you have any issues installing dependencies, check your node version against that defined in the `.nvmrc`. If you're using `nvm`, you can run `nvm use` before installing dependencies.
+
 5. Create a `.env` file and add the following variables. You can copy the contents of `.env.example` with `cat .env.example > .env`.
-  ```
-  # This default value is if you run our local docker-compose.yml file to create the database.
-  DATABASE_URL=postgresql://postgres:secret@127.0.0.1:5432/postgres
-  # Setup your GitHub ID & Secret on GitHub: https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps
-  # For development, make sure you set up this with a Homepage URL of http://localhost:3000/ and an Authorization callback URL of http://localhost:3000/api/auth
-  GITHUB_ID=YOUR_GITHUB_APP_ID
-  GITHUB_SECRET=YOUR_GITHUB_APP_SECRET
-  NEXTAUTH_URL=http://localhost:3000/api/auth
-  ```
-
-  For a more detailed how to guide on setting them up [go to the Environment Variables section](#environment-variables).
+
```

Comment:
No idea why its adding this line?  I didnt change this. Will look into my prettier settings as maybe I have a global config but it should be pulling from the repo config.

Reply:
Not a big deal I think anyway!